src/Components/Browser.JS/src/Platform/Circuits/AutoReconnectCircuitHandler.ts

```diff
@@ -0,0 +1,48 @@
+import { CircuitHandler } from "./CircuitHandler";
+export class AutoReconnectCircuitHandler implements CircuitHandler {
+  modal: HTMLDivElement;
+  intervalHandle: number | null;
+
+  constructor(private maxRetries: number = 3, private retryInterval: number = 3000) {
```

Comment:
Might have to tweak these numbers. Maybe 5 retries with 10 second delays?

Reply:
ten seconds might be a long delay depending on the UX